build.sbt

```diff
@@ -21,7 +23,7 @@ testOptions in ThisBuild += Tests.Argument(TestFrameworks.JUnit, "-v")
 
 transitiveClassifiers := Seq("sources", "javadoc")
 
-resolvers in ThisBuild += "NCSA Sonatype Releases" at "https://opensource.ncsa.illinois.edu/nexus/content/repositories/releases"
+resolvers in ThisBuild += "NCSA Sonatype Releases" at "https://daffodil.apache.org/nexus/content/repositories/releases"
```

Comment:
This url doesn't work.

Reply:
Actually, this line doesn't even belong in here anymore. Daffodil does not require dependencies from anything on NCSA. I'll remove this line.